Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7777,6 +7777,37 @@ def string_to_context_key(string):
         raise Exception('The key is not a string: {}'.format(string))
 
 
+def response_to_context(reponse_obj):
+    """Recursively creates a data dictionary where all key starts with capital letters.
+
+    :type reponse_obj: ``Any``
+    :param reponse_obj: The response object to update.
+
+    :return: A response with all keys (if there're any) starts with a capital letter.
+    :rtype: ``Any``
+    """
+    predefined_keys = {"id": "ID", "uuid": "UUID"}
```

Comment:
The developer using `response_to_context` should be able to send additional keys

Reply:
Great idea, adding the option with relevant tests

Reply:
Great idea, adding the option with relevant tests.